superset/db_engine_specs/base.py

```diff
@@ -252,6 +256,11 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
             types.DateTime(),
             GenericDataType.TEMPORAL,
         ),
+        (
+            re.compile(r"^time", re.IGNORECASE),
+            types.Time(),
+            GenericDataType.TEMPORAL,
+        ),
```

Comment:
If placed here, it will match `timestamp`

Reply:
Thanks. It's fixed